sdk/schema/validation.go

```diff
@@ -112,3 +112,27 @@ func CredentialReferencesInCredentialList(plugin Plugin) bool {
 	}
 	return true
 }
+
+func CredentialsAreUniquelyIdentifiableInsideAPlugin(plugin Plugin) bool {
```

Comment:
How about something like:
```suggestion
func PluginHasNoDuplicateCredentials(plugin Plugin) bool {
```
Or:
```suggestion
func NoDuplicateCredentials(plugin Plugin) bool {
```

Reply:
Done, thanks!